idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -235,7 +245,7 @@ def build(self):
         # Call UnitModel.build to build default attributes
         super(PackedColumnData, self).build()
 
```

Comment:
There should be some checking of general assumptions here - i.e. that the vapour property package has one phase named "Vap", the liquid properties have one phase names "Liq" and that there is at least 1 shared component between the two property packages.

See the Solvent Reboiler model for examples.